docs/src/literate-tutorials/computational_homogenization.jl

```diff
@@ -193,6 +193,21 @@ using Test #src
 # the [`FerriteGmsh`](https://github.com/Ferrite-FEM/FerriteGmsh.jl) package:
 
 using FerriteGmsh
+
+# Overload for specific elements in this tutorial.
+function FerriteGmsh.tofacesets(boundarydict::Dict{String,Vector}, elements::Vector{Triangle})
+    faces = Ferrite.facets.(elements)
+    facesets = Dict{String,Set{FaceIndex}}()
+    for (boundaryname, boundaryfaces) in boundarydict
+        facesettuple = Set{FaceIndex}()
```

Comment:
This is just a workaround since `Grid(cells, nodes; facesets)` works with a warning if `facetset` is not given. In FerriteGmsh this would be different. 
https://github.com/Ferrite-FEM/Ferrite.jl/blob/f8c67406bffa90afcaa92817fbdea7d161d11850/src/Grid/grid.jl#L390-L398